src/Objects.hs

```diff
@@ -10,42 +10,23 @@ import Data.Aeson
 import Control.Monad (forM)
 import Control.Applicative (optional)
 
+import GHC.Generics
+
 data Pod = Pod
   { name       :: Text
```

Comment:
`name` and `containers` are not top level in the json response.

They are nested under `metadata` and `spec` respectively.